app/app/src/main/java/org/omecproject/up4/impl/Up4DeviceManager.java

```diff
@@ -1440,7 +1452,10 @@ private void checkMeterStateAndReconcile() {
                         .forEach(m -> meterService.submit(m));
                 missingMeters.stream()
                         .map(m -> meterToMeterRequestForDevice(m, deviceId, true))
-                        .forEach(m -> meterService.submit(m));
+                        .forEach(m -> {
```

Comment:
you are missing the unexpected and the missing

Reply:
Now it should be ok